napari/resources/stylesheet.qss

```diff
@@ -991,3 +915,121 @@ QDockWidget::float-button:horizontal {
     left: 20px; top: 4;
     width: 14px;
 }
+
+/* ------------- QMainWindow --------- */
+
+QScrollBar {
+    border: none;
+    border-radius: 3px;
+    background: {{ foreground }};
+}
+
+QScrollBar:horizontal {
+    min-height: 14px;
+    max-height: 14px;
+    margin: 0px 16px;
+}
+
+QScrollBar:vertical {
+    /* min-width: 12px; */
+    max-width: 14px;
+    margin: 16px 0px;
+}
+
+QScrollBar::handle {
+    background: {{ highlight }};
+    border-radius: 3px;
+}
+
+QScrollBar::handle:horizontal {
+    min-width: 26px;
+}
+
+QScrollBar::handle:vertical {
+    min-height: 24px;
+}
+
+
+QScrollBar::add-line, QScrollBar::sub-line {
+    border: none;
+    border-radius: 3px;
+    background: {{ foreground }};
+    subcontrol-origin: margin;
+}
+
+QScrollBar::add-line:horizontal {
+    width: 13px;
+    subcontrol-position: right;
+}
+
+QScrollBar::sub-line:horizontal {
+    width: 13px;
+    subcontrol-position: left;
+}
+
+QScrollBar::add-line:vertical {
+    height: 13px;
+    subcontrol-position: top;
+}
+
+QScrollBar::sub-line:vertical {
+    height: 13px;
+    subcontrol-position: bottom;
+}
+
+QScrollBar::add-line:horizontal:pressed {
+    background: {{ highlight }};
+}
+
+QScrollBar::sub-line:horizontal:pressed {
+    background: {{ highlight }};
+}
+
+QScrollBar:left-arrow:horizontal {
+    image: url(":/icons/{{ folder }}/left_arrow.svg");
+}
+
+QScrollBar::right-arrow:horizontal {
+    image: url(":/icons/{{ folder }}/right_arrow.svg");
+}
+
+/* I know, it's weird... */
+QScrollBar:up-arrow:vertical {
+    image: url(":/icons/{{ folder }}/down_arrow.svg");
+}
+
+QScrollBar::down-arrow:vertical {
+    image: url(":/icons/{{ folder }}/up_arrow.svg");
+}
+
+QScrollBar::left-arrow,
+QScrollBar::right-arrow,
+QScrollBar::up-arrow,
+QScrollBar::down-arrow {
+    height: 9px;
+    width: 9px;
+}
+
+QtDims > QRangeSlider[last_used=true] {
+  qproperty-barColor: {{ secondary }};
+  qproperty-handleColor: {{ secondary }};
+  qproperty-handleBorderColor: {{ secondary }};
+}
+
+QtDims > QRangeSlider[last_used=false] {
+  qproperty-barColor: {{ highlight }};
+  qproperty-handleColor: {{ highlight }};
+  qproperty-handleBorderColor: {{ highlight }};
+}
+
+QtDims > QScrollBar::handle[last_used=false]:horizontal {
+    background: {{ highlight }};
+}
+
+QtDims > QScrollBar::handle[last_used=true]:horizontal {
+    background: {{ secondary }};
+}
+
+QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
+    background: none;
+}
```

Comment:
when you edit the stylesheet there never seems to be a new line added at the end, I know we aren't autolinting here, but how do you feel about it? Should we add?

Reply:
arg sorry!  I've grown so used to auto-formatting that I miss it on that file every time.  might as well add it.